astrometry/src/PixelMapCollection.cpp

```diff
@@ -135,6 +135,25 @@ PixelMapCollection::getParams() const {
   return p;
 }
 
+DVector
+PixelMapCollection::getParams(string mapName) const {
+  DVector p(parameterCount, 888.);
+  auto melpair = mapElements.find(mapName);
+  const MapElement& mel = melpair->second;
+  if (mel.isFixed) {
+    return DVector();
+  }
+  if (mel.atom) {
+    int nSub = mel.nParams;
+    DVector mparams = mel.atom->getParams().subVector(0, nSub);
+    DVector outP = mparams.subVector(20, 30);
```

Comment:
I don't really follow what this function is doing, and in particular where these magic numbers of 888, 20 and 30 came from.